salt/elife-dashboard/dashboard2.sls

```diff
@@ -0,0 +1,80 @@
+{% set app = pillar.elife_dashboard_2 %}
+{% set dash = pillar.elife_dashboard %}
+{% set user = pillar.elife.deploy_user.username %}
+
+
+#
+# db
+#
+
+
+
+#
+# configure
+#
+
+configure-{{ app.name }}:
+    file.managed:
+        - user: {{ user }}
+        - name: /srv/elife-dashboard/app.cfg
```

Comment:
I assume this means the Python code does a `../app.cfg` to read it

Reply:
Yes I assume its the parent diretory `CFG_PATH = os.path.join(BASE_DIR, '..')`.

Reply:
Good to go @giorgiosironi ?